### PR TITLE
Update cluster client replacement examples to Akka gRPC 0.8.4

### PR DESCRIPTION
**Pull request by johanandren**
Also:

 * Scala 2.13
 * sbt 1.3.9
 * Scalatest 3.1
 * Deprecated stream operators replaced

### CODE REVIEW

File: akka-sample-cluster-client-grpc-java/pom.xml [hunk 1]
```diff
@@ -18,37 +18,37 @@
   <dependencies>
     <dependency>
       <groupId>com.typesafe.akka</groupId>
-      <artifactId>akka-stream_2.12</artifactId>
+      <artifactId>akka-stream_2.13</artifactId>
```
Comment (ignasi35) [inline akka-sample-cluster-client-grpc-java/pom.xml:21]: I think it's a good moment to extract the scala binary version to a pom `<property>`.

File: akka-sample-cluster-client-grpc-java/project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.9
```
Comment (ignasi35) [inline akka-sample-cluster-client-grpc-java/project/build.properties:1]:
```suggestion
sbt.version=1.3.10
```

File: akka-sample-cluster-client-grpc-scala/project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.9
```
Comment (ignasi35) [inline akka-sample-cluster-client-grpc-scala/project/build.properties:1]:
```suggestion
sbt.version=1.3.10
```

File: akka-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala [hunk 4]
```diff
@@ -1,83 +1,95 @@
...
-import akka.actor.Props
-import akka.actor.Terminated
+import akka.actor.{
+  Actor,
+  ActorLogging,
+  ActorRef,
+  ActorSystem,
+  Props,
+  Terminated
+}
```
Comment (ignasi35) [inline akka-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala:11]: Wat? 
Should we use a wilcard import instead?

File: akka-sample-cluster-client-grpc-java/pom.xml (hunk 1 above)
Comment (johanandren) [reply akka-sample-cluster-client-grpc-java/pom.xml:21]: It doesn't really change often, and for Java users it's more of a "use this artifact" than think of it as a Scala version.

File: akka-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala (hunk 4 above)
Comment (johanandren) [reply akka-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala:11]: Not sure what my IntelliJ is smoking. I think a wildcard is bad in a sample though.

Comment (johanandren) [reply akka-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala]: Aha, it's after switching to community edition of IntelliJ. Reformatted with the Akka defaults now.

File: akka-sample-cluster-client-grpc-java/pom.xml (hunk 1 above)
Comment (ignasi35) [reply akka-sample-cluster-client-grpc-java/pom.xml:21]: Fair enough. I had https://github.com/lightbend/paradox/pull/429 in mind. LGTM

**Pull request merged**